AGENTS.md

```diff
@@ -0,0 +1,222 @@
+# AGENTS.md
+
+This file provides guidance to AI assistants (Claude, Gemini, etc).
+
+## Project Overview
+
+JobSet is a Kubernetes-native API for managing groups of Kubernetes Jobs as a unit, designed for distributed AI/ML training workloads (PyTorch, Jax, TensorFlow) and HPC applications (MPI). It is a Kubernetes SIG (Special Interest Group) project maintained under kubernetes-sigs.
+
```

Comment:
Might be good to add behaviour policy and context awareness: https://github.com/kubeflow/sdk/blob/main/AGENTS.md#agent-behavior-policy

Reply:
updated.